rosa_getting_started/rosa-getting-started-workflow.adoc

```diff
@@ -13,11 +13,10 @@ Follow this workflow to set up and access {product-title} (ROSA) clusters:
 . xref:../rosa_getting_started/rosa-config-aws-account.adoc#rosa-config-aws-account[Configure your AWS account].
 . xref:../rosa_getting_started/rosa-installing-rosa.adoc#rosa-installing-rosa[Install ROSA].
 . xref:../rosa_getting_started/rosa-creating-cluster.adoc#rosa-creating-cluster[Create a ROSA cluster] or xref:../rosa_getting_started/rosa-aws-privatelink-creating-cluster.adoc#rosa-aws-privatelink-creating-cluster[Create a ROSA cluster using AWS PrivateLink].
-. xref:../rosa_getting_started/rosa-config-identity-providers.adoc#rosa-config-identity-providers[Configure identity providers].
 . xref:../rosa_getting_started/rosa-accessing-cluster.adoc#rosa-accessing-cluster[Access a cluster].
 
 == Additional resources
-
+* xref:../rosa_getting_started/rosa-config-identity-providers.adoc#rosa-config-identity-providers[Configure identity providers using the OCM console]
```

Comment:
The topic title is "Configuring" rather than "Configure" - might be good to use that esp since the others below are "Deleting" (w/gerund)